### DISCUSSION TITLE
2022/09/20/csp-2023-rp-plus-plus/

### DISCUSSION THREAD

**Discussion by giscus** [DeCalvin2006/blog#5] [General]
# 2022/09/20/csp-2023-rp-plus-plus/

Day -1 lt提前说了要坐车润。但是我爸准备开车带我去（（（ 啥也没干，颓了一天……  Day 0. 上午做了LGR-SCP-S2022的题，感觉根本做不动啊，是不是要寄了））） 下午回家打ADOFAI。这个T4-X卡了好长时间了QwQ，只能刷下前面的FP（Full Perfect）了  T2:4FP，3P T1:1FP, 6P T3:啥也没刷到…… T4:在不等边三角形寄了好长时间……

https://decalvin.netlify.app/2022/09/20/csp-2023-rp-plus-plus/

**Comment by deyerong** [comment]
# ヾ(≧▽≦*)o
## ψ(｀∇´)ψψ(｀∇´)ψψ(｀∇´)ψ

**Comment by DeCalvin2006** [comment]
QwQ

________________________________
发件人: 李闻焌 ***@***.***>
发送时间: 2022年9月23日 09:00
收件人: DeCalvin2006/blog ***@***.***>
抄送: Subscribed ***@***.***>
主题: Re: [DeCalvin2006/blog] 2022/09/20/csp-2023-rp-plus-plus/ (Discussion #5)


ヾ(≧▽≦*)o
ψ(｀∇´)ψψ(｀∇´)ψψ(｀∇´)ψ
